src/super_gradients/training/datasets/pose_estimation_datasets/coco_utils.py

```diff
@@ -1,11 +1,33 @@
+from enum import Enum
+
 import numpy as np
 from pycocotools.coco import COCO
+from tqdm import tqdm
 
 from super_gradients.common.abstractions.abstract_logger import get_logger
 
 logger = get_logger(__name__)
 
-__all__ = ["check_keypoints_outside_image", "check_for_duplicate_annotations", "make_keypoints_outside_image_invisible", "remove_duplicate_annotations"]
+__all__ = [
+    "CrowdAnnotationActionEnum",
+    "check_keypoints_outside_image",
+    "check_for_duplicate_annotations",
+    "make_keypoints_outside_image_invisible",
+    "remove_duplicate_annotations",
+    "remove_crowd_annotations",
+    "remove_samples_with_crowd_annotations",
+]
+
+
+class CrowdAnnotationActionEnum(str, Enum):
+    """
+    Enum that contains possible actions to take for crowd annotations.
+    """
+
+    DROP_SAMPLE = "drop_sample"
+    DROP_ANNOTATION = "drop_annotation"
+    MASK_AS_NORMAL = "mask_as_normal"
+    NO_ACTION = "no_action"
```

Comment:
Is this used? I didn't fine any reference

Reply:
It is used in COCOPoseEstimationDataset which is not yet in PR (This enum slipped by accident to this PR)

Reply:
Ok It's all right we can keep in